R/utils_gcp.R

```diff
@@ -0,0 +1,102 @@
+# Semi-automated tests of Google Cloud Storage integration live in tests/gcp/. # nolint
+# These tests should not be fully automated because they
+# automatically create S3 buckets and upload data,
+# which could put an unexpected and unfair burden on
+# external contributors from the open source community.
+# nocov start
+gcp_gcs_exists <- function(
+  key, 
+  bucket = googleCloudStorageR::gcs_get_global_bucket(), 
```

Comment:
If you would address lints like these, that would be great. I'm pretty pedantic about code style for `targets`, and I try to follow most of `lintr`'s standards.

Reply:
I'm not sure what the issue is here, I was trying to make sure the second line wasn't over 80chars

Reply:
I added `gcp_gcs_bucket()` to stop the import making long lines

Reply:
Mainly the lints seem to be from trailing whitespace at the end of a line. You can run the linters locally with `lintr::lint_package()`.